security/src/main/java/io/micronaut/security/token/cookie/CookieLoginHandler.java

```diff
@@ -107,7 +107,7 @@ public MutableHttpResponse<?> loginSuccess(Authentication authentication, HttpRe
     public MutableHttpResponse<?> loginFailed(AuthenticationResponse authenticationFailed, HttpRequest<?> request) {
```

Comment:
The JavaDoc for this method needs updating to reflect the new behavior.

Reply:
good catch. I have updated the javadoc